poetry.lock

```diff
@@ -1,9 +1,10 @@
-# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry and should not be changed by hand.
```

Comment:
I think your version of poetry might be old, which explains why this line does not show a version and also why so many lines of the lock file changed. 

Can you make sure it's the latest version and then regenerate the file?